lib/src/action.dart

```diff
@@ -65,6 +67,7 @@ class Action<T> implements Function {
   /// Cancel all subscriptions that exist on this [Action] as a result of
   /// [listen] being called. Useful when tearing down a flux cycle in some
   /// module or unit test.
+  @Deprecated('Use (and await) dispose() instead.')
```

Comment:
In order to get the same effect, `dispose(...)` has to be awaited because it is async.